anchore_engine/services/policy_engine/engine/policy/gates/vulnerabilities.py

```diff
@@ -39,12 +39,16 @@ class VulnerabilityMatchTrigger(BaseTrigger):
     severity = EnumStringParameter(name='severity', example_str='high', description='Severity to compare against.', enum_values=SEVERITY_ORDERING, is_required=True, sort_order=3)
     fix_available = BooleanStringParameter(name='fix_available', example_str='true', description='If present, the fix availability for the vulnerability record must match the value of this parameter.', is_required=False, sort_order=4)
     vendor_only = BooleanStringParameter(name='vendor_only', example_str='true', description='If True, an available fix for this CVE must not be explicitly marked as wont be addressed by the vendor', is_required=False, sort_order=5)
+    days_past_creation = IntegerStringParameter(name='days_past_creation', example_str='7', description='If provided, this CVE must be older than the days provided to trigger.', is_required=False, sort_order=6)
```

Comment:
For consistency with our other gates and trigger parameter names, can you change the name field (the var name is fine) in this to 'max_days_since_creation'. We use 'max_days_since_sync' in our feeds timeliness check which is similar semantically, so I want to keep the naming similar.

Reply:
Thank you for the review. I updated this to match and went ahead and updated the variable names too so everything matches.

Reply:
I think there was some confusion on the name, I wasn't proposing the exact same name, but something like 'max_days_since_creation' for this particular parameter, which follows a similar naming scheme to the other feed sync param, but is clear in this vuln context. Apologies for that lack of clarity. You can just push a new commit and I'll merge-squash it down. That makes it easier to review here as well because github will notice the change and map it to the comment. Overall, though, this is great work and super useful stuff. Thanks!

Reply:
You were clear and I find/replaced the wrong thing. fix coming soon.

Reply:
Thanks! looks good.